blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -65,6 +65,7 @@ export class RunRecord extends Record({
     state: null,
     type: null,
     commitId: null,
+    actions: null,
 }) {
     isQueued() {
```

Comment:
@scherler filter just removes objects from array, .map would return just the _clsss.

```
getActions(class) {
  if(!class) {
    return this.actions;
  }
  return this.actions.filter(action => action._class == class);
}
```